### ISSUE TITLE
Error: yt-dlp v2024.7.9 Returns "The following content is not available on this app" for All Videos

### ISSUE THREAD

**Issue by Rheetiks** [yt-dlp/yt-dlp#10482]
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that I will be **blocked** if I *intentionally* remove or skip any mandatory\* field

### Checklist

- [X] I'm reporting a bug unrelated to a specific site
- [X] I've verified that I have **updated yt-dlp to nightly or master** ([update instructions](https://github.com/yt-dlp/yt-dlp#update-channels))
- [X] I've checked that all provided URLs are playable in a browser with the same IP and same login details
- [X] I've checked that all URLs and arguments with special characters are [properly quoted or escaped](https://github.com/yt-dlp/yt-dlp/wiki/FAQ#video-url-contains-an-ampersand--and-im-getting-some-strange-output-1-2839-or-v-is-not-recognized-as-an-internal-or-external-command)
- [X] I've searched [known issues](https://github.com/yt-dlp/yt-dlp/issues/3766) and the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues **including closed ones**. DO NOT post duplicates
- [x] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)

### Provide a description that is worded well enough to be understood

After initially using yt-dlp version 2024.4.9, I encountered an issue where only some videos from YouTube displayed the error message: "The following content is not available on this app. Watch on the latest version of YouTube." However, after updating to yt-dlp version 2024.7.9, the issue has worsened, and now all video download attempts result in the same error. Even after downgrading back to version 2024.4.9, the issue persists, indicating that the problem is not limited to a specific version but might be due to recent changes in YouTube’s site or API that affect how yt-dlp handles video extraction.

### Provide verbose output that clearly demonstrates the problem

- [X] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [X] If using API, add `'verbose': True` to `YoutubeDL` params instead
- [X] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

### Complete Verbose Output

```shell
[debug] Command-line config: ['-v', 'https://www.youtube.com/shorts/ABwOK9YrMJw']
[debug] Encodings: locale UTF-8, fs utf-8, pref UTF-8, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version stable@2024.07.09 from yt-dlp/yt-dlp [ff0779267] (pip)
[debug] Python 3.10.14 (CPython x86_64 64bit) - Linux-5.15.153.1-1.cm2-x86_64-with-glibc2.31 (OpenSSL 1.1.1w  11 Sep 2023, glibc 2.31)
[debug] exe versions: ffmpeg 4.3.6-0, ffprobe 4.3.6-0
[debug] Optional libraries: Cryptodome-3.20.0, brotli-1.1.0, certifi-2024.02.02, mutagen-1.47.0, requests-2.32.3, sqlite3-3.34.1, urllib3-1.26.18, websockets-12.0
[debug] Proxy map: {}
[debug] Request Handlers: urllib, requests, websockets
[debug] Loaded 1810 extractors
[youtube] Extracting URL: https://www.youtube.com/shorts/ABwOK9YrMJw
[youtube] ABwOK9YrMJw: Downloading webpage
[youtube] ABwOK9YrMJw: Downloading ios player API JSON
[youtube] ABwOK9YrMJw: Downloading android player API JSON
ERROR: [youtube] ABwOK9YrMJw: The following content is not available on this app. Watch on the latest version of YouTube.
  File "/tmp/8dca58883ae455f/antenv/lib/python3.10/site-packages/yt_dlp/extractor/common.py", line 734, in extract
    ie_result = self._real_extract(url)
  File "/tmp/8dca58883ae455f/antenv/lib/python3.10/site-packages/yt_dlp/extractor/youtube.py", line 4182, in _real_extract
    self.raise_no_formats(reason, expected=True)
  File "/tmp/8dca58883ae455f/antenv/lib/python3.10/site-packages/yt_dlp/extractor/common.py", line 1251, in raise_no_formats
    raise ExtractorError(msg, expected=expected, video_id=video_id)
```

**Comment by bashonly** [comment]
Duplicate of #10108

Your IP has been blocked by Youtube

**Comment by Rheetiks** [comment]
is there any way to prevent this like how much should I wait before downloading next video

**Comment by bashonly** [comment]
It's not known what exactly triggers an IP ban like this.

You could try using some options like these:
```
--sleep-requests 1.25 --min-sleep-interval 60 ---max-sleep-interval 90
```

Youtube does seem to be targeting data center IP addresses with these bans, so if you're using one, the above options may not even help

**Comment by bashonly** [comment]
> [youtube] ABwOK9YrMJw: Downloading webpage
> [youtube] ABwOK9YrMJw: Downloading ios player API JSON
> [youtube] ABwOK9YrMJw: Downloading android player API JSON

actually, something is off here: why is android being requested? this was removed from the defaults a while ago. did you edit any source code files? did you edit this log?

**Comment by bashonly** [comment]
> [debug] Loaded 1810 extractors

yeah this is wrong. you are actually on an old version of yt-dlp. completely uninstall and re-install

**Comment by Rheetiks** [comment]
No I haven't changed any source, and I am currently on the latest version of yt-dlp 2024.7.9

@bashonly

**Comment by bashonly** [comment]
> completely uninstall and re-install

**Comment by Rheetiks** [comment]
``[youtube] Extracting URL: https://www.youtube.com/shorts/mgBbAwviTNY
[youtube] mgBbAwviTNY: Downloading `webpage`
[youtube] mgBbAwviTNY: Downloading ios player API JSON
ERROR: [youtube] mgBbAwviTNY: The following content is not available on this app.. Watch on the latest version of YouTube.
(antenv) root@fb-automat_250cd29e5c:/tmp/8dca58883ae455f# yt-dlp -v https://www.youtube.com/shorts/mgBbAwviTNY
[debug] Command-line config: ['-v', 'https://www.youtube.com/shorts/mgBbAwviTNY']
[debug] Encodings: locale UTF-8, fs utf-8, pref UTF-8, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version stable@2024.07.16 from yt-dlp/yt-dlp [89a161e8c] (pip)
[debug] Python 3.10.14 (CPython x86_64 64bit) - Linux-5.15.153.1-1.cm2-x86_64-with-glibc2.31 (OpenSSL 1.1.1w  11 Sep 2023, glibc 2.31)
[debug] exe versions: ffmpeg 4.3.6-0, ffprobe 4.3.6-0
[debug] Optional libraries: Cryptodome-3.20.0, brotli-1.1.0, certifi-2024.02.02, mutagen-1.47.0, requests-2.32.3, sqlite3-3.34.1, urllib3-1.26.18, websockets-12.0
[debug] Proxy map: {}
[debug] Request Handlers: urllib, requests, websockets
[debug] Loaded 1829 extractors
[youtube] Extracting URL: https://www.youtube.com/shorts/mgBbAwviTNY
[youtube] mgBbAwviTNY: Downloading webpage
WARNING: [youtube] Webpage contains broken formats (poToken experiment detected). Ignoring initial player response
[youtube] mgBbAwviTNY: Downloading ios player API JSON
[youtube] mgBbAwviTNY: Downloading player 65e3708e
[youtube] mgBbAwviTNY: Downloading web player API JSON
ERROR: [youtube] mgBbAwviTNY: The following content is not available on this app.. Watch on the latest version of YouTube.
  File "/tmp/8dca58883ae455f/antenv/lib/python3.10/site-packages/yt_dlp/extractor/common.py", line 740, in extract
    ie_result = self._real_extract(url)
  File "/tmp/8dca58883ae455f/antenv/lib/python3.10/site-packages/yt_dlp/extractor/youtube.py", line 4267, in _real_extract
    self.raise_no_formats(reason, expected=True)
  File "/tmp/8dca58883ae455f/antenv/lib/python3.10/site-packages/yt_dlp/extractor/common.py", line 1263, in raise_no_formats
    raise ExtractorError(msg, expected=expected, video_id=video_id)
    
    
    
    
    
    still the error is same but the logs has changed a bit
    
    @bashonly

**Comment by Rheetiks** [comment]
i think its the issue with ip only because if i try downloading the same video by passing proxy the videos starts downloading

@bashonly
